src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -0,0 +1,114 @@
+
+{% set choices_flat = choices
+    | map(choice => choice.choices is defined ? choice.choices : [choice])
+    | reduce((carry, choices) => carry|merge(choices), [])
+%}
+{% set preferred_choices = preferred_choices|default([]) %}
+{% set min_search_items = min_search_items|default(5) %}
+{% set custom_form = custom_form is defined ? custom_form : true %}
+{% set translation_domain = translation_domain|default(false) %}
+{% set value = value is defined ? value : null %}
+{% set multiple = multiple|default(false) %}
+{% set no_items = choices_flat|length == 0 %}
+
+{% macro get_translated_label(label, translation_domain) %}
+    {{ translation_domain is same as(false) ? label : label|trans({}, translation_domain) }}
+{% endmacro %}
+
+{% macro get_translated_placeholder(placeholder, translation_domain) %}
+    {% if placeholder is defined %}
```

Comment:
Can you pass a variable that is not defined (in this case `placeholder`) to a macro?

Reply:
You cannot, as discussed in private :)